code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -932,7 +932,11 @@ GLOBAL_LIST_INIT(robot_verbs_default, list(
 			to_chat(src, "<span class='warning'>ERRORERRORERROR</span>")
 			to_chat(src, "<b>Obey these laws:</b>")
 			laws.show_laws(src)
-			to_chat(src, "<span class='boldwarning'>ALERT: [M.real_name] is your new master. Obey your new laws and [M.p_their()] commands.</span>")
+			if(!mmi.syndiemmi)
+				to_chat(src, "<span class='boldwarning'>ALERT: [M.real_name] is your new master. Obey your new laws and [M.p_their()] commands.</span>")
+			if(mmi.syndiemmi && mmi.master_uid)
```

Comment:
For clarity: 

```suggestion
			else if(mmi.syndiemmi && mmi.master_uid)
```

But also, what about the rest of the else branch? why does nothing happen when the mmi is syndicate but (somehow?) has no master?...
If that's a "this should never happen" situation, then it'd be clearer if this was either not handled at all (i.e. remove the `&& mmi.master_uid` check) or spawn a descriptive runtime so that a future someone can debug what the hell went wrong to allow this.

Reply:
My logic for not adding a message here was that 1. This shouldnt happen, but if it does it's not gamebreaking and doesnt need a specific runtime 2. If it does, the cyborg goes from being loyal to the syndicate to loyal to the syndicate and emagged, which I dont think warrants a warning message (As opposed to two traitors who might have conflicting objectives)

Reply:
I'll add an else case message anyway to clarify